dataproc/quickstart/quickstart.py

```diff
@@ -15,25 +15,24 @@
 # limitations under the License.
 
 # [START dataproc_quickstart]
+"""This quickstart sample walks a user through creating a Cloud Dataproc
+    cluster, submitting a PySpark job from Google Cloud Storage to the
+    cluster, reading the output of the job and deleting the cluster, all
+    using the Python client library.
+
+    Usage:
+        python3 quickstart.py --project_id <PROJECT_ID> --region <REGION> \
```

Comment:
Is this a python3 only sample? I _think_ the official policy is we are still supporting 2.7 atm.